docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
+
+services:
+  cassandra:
+    ports:
+      - 9042:9042
```

Comment:
I normally tend to not add extra indentation to sequences. This helps to deal with nested things better.